db/migrate/20250912000001_populate_pulp_prn_fields.rb

```diff
@@ -0,0 +1,389 @@
+    # add_index :katello_generic_content_units, :pulp_prn, unique: true
+    # add_index :katello_module_streams, :pulp_prn, unique: true
+    # add_index :katello_docker_manifest_lists, :pulp_prn, unique: true
+    # add_index :katello_docker_manifests, :pulp_prn, unique: true
+    # add_index :katello_docker_tags, :pulp_prn, unique: true
+    # add_index :katello_rpms, :pulp_prn, unique: true
+    # add_index :katello_srpms, :pulp_prn, unique: true
+    # add_index :katello_files, :pulp_prn, unique: true
+    # add_index :katello_package_groups, :pulp_prn, unique: true
+    # add_index :katello_debs, :pulp_prn, unique: true
+  end
+
+  def down
+    # # Remove unique indexes before clearing data
+    # remove_index :katello_ansible_collections, :pulp_prn
+    # remove_index :katello_generic_content_units, :pulp_prn
+    # remove_index :katello_module_streams, :pulp_prn
+    # remove_index :katello_docker_manifest_lists, :pulp_prn
+    # remove_index :katello_docker_manifests, :pulp_prn
+    # remove_index :katello_docker_tags, :pulp_prn
+    # remove_index :katello_rpms, :pulp_prn
+    # remove_index :katello_srpms, :pulp_prn
+    # remove_index :katello_files, :pulp_prn
+    # remove_index :katello_package_groups, :pulp_prn
+    # remove_index :katello_debs, :pulp_prn
+    #
+    # # Remove NOT NULL constraints before clearing data
+    # change_column_null :katello_content_guards, :pulp_prn, true
+    # change_column_null :katello_repository_references, :repository_prn, true
+    # change_column_null :katello_ansible_collections, :pulp_prn, true
+    # change_column_null :katello_generic_content_units, :pulp_prn, true
+    # change_column_null :katello_module_streams, :pulp_prn, true
+    # change_column_null :katello_docker_manifest_lists, :pulp_prn, true
+    # change_column_null :katello_docker_manifests, :pulp_prn, true
+    # change_column_null :katello_docker_tags, :pulp_prn, true
+    # #change_column_null :katello_errata, :pulp_prn, true
+    # change_column_null :katello_rpms, :pulp_prn, true
+    # change_column_null :katello_srpms, :pulp_prn, true
+    # change_column_null :katello_files, :pulp_prn, true
+    # change_column_null :katello_package_groups, :pulp_prn, true
+    # change_column_null :katello_debs, :pulp_prn, true
+
+    # Clear all PRN fields
+    execute "UPDATE katello_content_guards SET pulp_prn = NULL"
+    execute "UPDATE katello_repositories SET remote_prn = NULL, publication_prn = NULL, version_prn = NULL"
+    execute "UPDATE katello_ansible_collections SET pulp_prn = NULL"
+    execute "UPDATE katello_generic_content_units SET pulp_prn = NULL"
+    execute "UPDATE katello_module_streams SET pulp_prn = NULL"
+    execute "UPDATE katello_docker_manifest_lists SET pulp_prn = NULL"
+    execute "UPDATE katello_docker_manifests SET pulp_prn = NULL"
+    execute "UPDATE katello_docker_tags SET pulp_prn = NULL"
+    execute "UPDATE katello_errata SET pulp_prn = NULL"
```

Comment:
I realized the katello_errata table shouldn't have a PRN value associated with it.